Data/alphabets/alphabet.latin.xml

```diff
@@ -128,6 +129,7 @@ Any word that can be written using one of the Latin ISO-8859 character sets (ISO
 <encoding type="Western"/>
 <palette>European/Asian</palette>
 <train>training_frenchC_FR.txt</train>
+<langcode>fr</langcode>
```

Comment:
Note: The "all Latin scripts" alphabet doesn't seem like it belongs in the same file as Latin the language.